flake.nix

```
@@ -3,26 +3,19 @@
 
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
 
-  outputs = {
-    self,
-    nixpkgs,
-    ...
-  }: let
-    systems = ["aarch64-darwin" "aarch64-linux" "x86_64-darwin" "x86_64-linux"];
-    forEachSystem = fn: lib.genAttrs systems (system: fn (import nixpkgs {inherit system;}));
-    inherit (nixpkgs) lib;
-  in {
-    checks = forEachSystem (pkgs: self.packages.${pkgs.system});
-    packages = forEachSystem (pkgs: lib.filterAttrs (_: v: lib.isDerivation v) (pkgs.callPackage ./nix { }) // { whiskers = throw "This package has been moved to x `github:catppuccin/whiskers`"; } );
-    overlays.default = final: prev: {
-      catppuccin-catwalk = (prev.callPackage ./nix {}).catwalk;
-      catppuccin-whiskers = throw "This package has been moved to x `github:catppuccin/whiskers`";
+  outputs =
+    { nixpkgs, ... }:
+    let
+      forAllSystems =
+        function:
+        nixpkgs.lib.genAttrs nixpkgs.lib.systems.flakeExposed (
+          system: function nixpkgs.legacyPackages.${system}
+        );
+    in
+    {
+      packages = forAllSystems (_: {
+        whiskers = throw "This package has been moved to `github:catppuccin/whiskers`";
+        catwalk = throw "This package has been moved to `github:catppuccin/catwalk`";
+      });
```

If we are trying not to break apis we should keep the overlay
```suggestion
      });
      
      overlays.default = _: _: {
        catppuccin-catwalk = throw "This package has been moved to `github:catppuccin/catwalk`";
         catppuccin-whiskers = throw "This package has been moved to `github:catppuccin/whiskers`";
};
```